src/Psalm/Codebase.php

```diff
@@ -1422,12 +1423,19 @@ public function getCompletionItemsForClassishThing(string $type_string, string $
                             null,
                             (string)$method_storage->visibility,
                             $method_storage->cased_name,
-                            $method_storage->cased_name . (count($method_storage->params) !== 0 ? '($0)' : '()'),
+                            null,
```

Comment:
Should this be `$method_storage->cased_name`? I actually have no idea, it just seems right at first sight.

Reply:
This used to set the `insertText` member using a placeholder when there were any parameters. Now, it is initialized to `null` in the constructor, and later given a value based on the parameters and whether placeholders are supported. Does that answer your question? I'm going to go ahead and resolve this already.